website/source/docs/enterprise/autopilot/index.html.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: "docs"
+page_title: "Nomad Enterprise Advanced Autopilot"
+sidebar_current: "docs-enterprise-autopilot"
+description: |-
+  Nomad Enterprise supports Advanced Autopilot capabilities which enable fully 
+  automated server upgrades, higher throughput for reads and scheduling, and hot 
+  server failover on a per availability zone basis.
+---
+
+# Nomad Enterprise Advanced Autopilot
+
+Nomad Enterprise supports Advanced Autopilot capabilities which enable fully
+automated server upgrades, higher throughput for reads and scheduling, and hot
+server failover on a per availability zone basis. See the sections below for 
+additional details on each of these capabilities. 
+
+* **Automated Upgrades:** Advanced Autopilot enables an upgrade pattern that 
+allows operators to deploy a complete cluster of new servers and then just wait 
```

Comment:
"and then simply wait for the server to complete"

Reply:
Perhaps write a quick summary (or link to) the open source requirement for upgrades.